### ISSUE TITLE
Navigation buttons are cutoff in landscape

### ISSUE THREAD

**Issue by drewvolz** [StoDevX/AAO-React-Native#2483]
<img src="https://user-images.githubusercontent.com/5240843/38164589-17b17dcc-34c4-11e8-9d35-b1f11e92ed67.png" width=450 />

**Comment by rye** [comment]
I can't reproduce on Android.  Could it be something iPhone X-related?

**Comment by hawkrives** [comment]
@drewvolz Would you please bisect based on react-navigation updates? My padding things from when we first ported to react-navigation may be unnecessary now, ~50-odd versions later.

**Comment by drewvolz** [comment]
Let's look at this. `buttonStyle` is `undefined` for both the `EditHomeButton` and `OpenSettingsButton` functions. We could pass in a style there but we do not. It is an optional prop, which means that's fine.

Look at `commonStyles` which is only used by `EditHomeButton`.

https://github.com/StoDevX/AAO-React-Native/blob/45daa8fc0590a7bbe9d74bc9708cea2d1d7c775b/source/views/components/nav-buttons/edit-home.js#L10

https://github.com/StoDevX/AAO-React-Native/blob/45daa8fc0590a7bbe9d74bc9708cea2d1d7c775b/source/views/components/nav-buttons/styles.js#L9-L22

It looks like `OpenSettingsButton` is receiving padding styles directly from within the file itself rather than from `commonStyles`.

https://github.com/StoDevX/AAO-React-Native/blob/45daa8fc0590a7bbe9d74bc9708cea2d1d7c775b/source/views/components/nav-buttons/open-settings.js#L37-L42

In summary, `paddingVertical` on the `open-settings` and `edit-home` buttons are applied via two different methods. But both of those styles are probably unnecessary now. I've taken them out in a test and it fixes the cutoff issue.
